wp-content/themes/core/blocks/outermost/socialsharing/style.pcss

```diff
@@ -0,0 +1,43 @@
+/**
+ * Styles specific to this block
+ */
+
+body {
+	--ssb-button-color: var(--wp--preset--color--base-black);
+	--ssb-button-hover-color: var(--wp--preset--color--base-white);
+	--ssb-button-background-color: var(--wp--preset--color--base-white);
+	--ssb-button-background-hover-color: var(--wp--preset--color--base-black);
+	--ssb-button-border-color: var(--wp--preset--color--base-black);
+	--ssb-button-border-hover-color: var(--wp--preset--color--base-white);
+	--ssb-button-border-size: 1px;
+}
+
+.wp-block-outermost-social-sharing,
+.editor-styles-wrapper .block-editor-block-list__layout.is-root-container .wp-block-outermost-social-sharing {
+	background: transparent;
+	transition: var(--transition);
+}
+
+.wp-block-outermost-social-sharing:not(.is-style-logos-only) .outermost-social-sharing-link {
```

Comment:
Any chance we can nest this and the selectors above to reduce the repetition?

Reply:
Since this is a different element, I'd almost prefer them to be separated. Maybe just use a block comment to denote that this is a different element within the block. Again, unless this is a case where we need a more complex selector to override other styling, then this should be fine. If we do need a more complex selector then nesting maybe does make more sense.